src/storage/src/hummock/mod.rs

```diff
@@ -243,15 +237,15 @@ impl HummockStorage {
     }
 
     pub fn hummock_meta_client(&self) -> &Arc<dyn HummockMetaClient> {
```

Comment:
Is `hummock_meta_client`, `options` and `sstable_store` still used externally? If not, we may remove these methods.

Reply:
ok, hummock_meta_client is deprecated, but the other are still using in `test` or `tool`